web/models/user_from_auth.ex

```diff
@@ -20,12 +28,22 @@ defmodule UserFromAuth do
     else
       name = [auth.info.first_name, auth.info.last_name]
       |> Enum.filter(&(&1 != nil and &1 != ""))
-      if length(name) == 0, do: auth.info.nickname, else: name = Enum.join(name, " ")
+
+      cond do
+        length(name) == 0 -> auth.info.nickname
+        true -> Enum.join(name, " ")
+      end
     end
   end
 
-  defp validate_password(%Credentials{other: %{password: ""}}), do: {:error, "Password required"}
-  defp validate_password(%Credentials{other: %{password: pw, password_confirmation: pw}}), do: :ok
-  defp validate_password(%Credentials{other: %{password: _}}), do: { :error, "Passwords do not match" }
-  defp validate_password(_), do: {:error, "Password Required"}
+  defp validate_pass(%{other: %{password: ""}}) do
+    {:error, "Password required"}
+  end
+  defp validate_pass(%{other: %{password: pw, password_confirmation: pw}}) do
```

Comment:
@hassox are you okay with this change?  With `validate_password` and `%Credentials` it was impossible to get under 80.